kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -259,6 +259,46 @@ sealed abstract class Stream[V, -S]:
         ))
     end changes
 
+    /** Transforms the stream by regrouping elements into chunks of the specified size.
+      *
+      * This operation maintains the order of elements while potentially redistributing them into new chunks. Smaller chunks may occur in
+      * two cases:
+      *   - When there aren't enough remaining elements to form a complete chunk
+      *   - When the input stream emits an empty chunk
+      *
+      * @param chunkSize
+      *   The target size for each chunk. Must be positive - negative values will be treated as 1.
+      * @return
+      *   A new stream with elements regrouped into chunks of the specified size
+      */
+    def rechunk(chunkSize: Int)(using tag: Tag[Emit[Chunk[V]]], frame: Frame): Stream[V, S] =
+        Stream.sized[V, S](chunkSize)(cs =>
```

Comment:
I find it a bit confusing that this method only transforms the `chunkSize`. This kind of API pattern typically transforms the computation as well. Could it be a method that returns `Int` instead?

Reply:
Also, how about using `chunkSize` instead of `cs` for clarity?

Reply:
Removing the function for now.